.circleci/config.yml

```diff
@@ -39,7 +39,7 @@ jobs:
   docs:
     <<: *common
     docker:
-      - image: cimg/python:3.7
```

Comment:
We have the convention currently to leave this as the lowest supported version, but feel free to take it or leave it

Reply:
no problem, reverted